sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -577,21 +633,21 @@ public String apply(String input) {
             jobIdTokenCollection.apply(
                 "RunCreateJob",
                 ParDo.of(
-                        new DoFn<String, String>() {
-                          @ProcessElement
-                          public void processElement(ProcessContext c) throws Exception {
-                            String jobUuid = c.element();
-                            BigQuerySourceBase source = createSource(jobUuid);
-                            String schema =
-                                BigQueryHelpers.toJsonString(
-                                    source.getSchema(c.getPipelineOptions()));
-                            c.output(tableSchemaTag, schema);
-                            List<ResourceId> files = source.extractFiles(c.getPipelineOptions());
-                            for (ResourceId file : files) {
-                              c.output(file.toString());
-                            }
-                          }
-                        })
+                    new DoFn<String, String>() {
+                      @ProcessElement
```

Comment:
It would be good to put the formatting changes into a separate commit (first commit in the chain ideally) to avoid polluting the diff.

Reply:
ugh yeah, sadly a lot of the reformatting was intellij imposing its coding style on the code.  I can try to un-format it.